docs/conf.py

```diff
@@ -18,15 +18,15 @@
 sys.path.insert(0, os.path.abspath("./ext"))
 sys.path.insert(0, os.path.abspath("../"))
 
-from screenpy.__version__ import __version__  # noqa: need the path first
+from screenpy.__version__ import __version__, __author__  # noqa: need the path first
 
 autodoc_mock_imports = ["hamcrest", "typing_extensions"]
 
 # -- Project information -----------------------------------------------------
 
 project = "ScreenPy"
-copyright = "2019, Perry Goy"
-author = "Perry Goy"
+copyright = f"2019, {__author__}"
```

Comment:
Should we import  `__copyright__`?

Reply:
That's not a bad idea. Also, can't we add `coauthors` or something? I feel bad that it's just my name on here, we should get yours and Gabe's too.

Reply:
Haha maybe it should be `ScreenPyHQ` as the copyright holder. Honestly i don't know much about copyrights.

I just meant that there's a "contributors" part for PyPi right? It's about time y'all had your names on this thing publicly, too.

Reply:
> maybe it should be ScreenPyHQ 

That's a cool idea.  There may be legal implications to assigning copyright to an entity which might require something like an LLC registration.  This will require legal advice.  For now, I'm going to just leave it under your name  as I wouldn't want to jeopardize your copyright holding.